.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+sudo: false
+node_js:
+  - 6
+env:
+  - NODE_ENV="test"
+script:
+  - npm test
+  #- npm run enforce
+  - npm run lint
```

Comment:
can you change these `npm run`s to `yarn`? so itd be

```
- yarn test
# - yarn enforce
- yarn lint
```

yarn doesnt need the `run` cause its smarter than your average bear